apps/webapp/README.md

```diff
@@ -0,0 +1,20 @@
+# Starkbot website
+
+Manage your keys from that website.
+
+### Dev setup
+
+Install `npm` and `nodejs` packages according to your distribution.
+create a *.env* file and add the following lines.
+
+```
+JWT_KEY = <KEY TO SIGN JWT TOKEN>
+RST_KEY = <KEY TO SIGN THE RESET TOKEN>
+API_URL = "http://localhost:3000/api"
+```
+
+Take care of choosing great keys.
+
+Run `pnpm dev` in the webapp directory
```

Comment:
Nope ! pnpm dev should be run in root directory since we're in a monorepo project. You should add dependencies to package.json at root and add a project.json in webapp to setup the target.

Reply:
See project.json in discrod:bot to understand how it works